test/test_quantized_conv.py

```diff
@@ -6,12 +6,12 @@
 import numpy as np
```

Comment:
btw we probably want to merge this into test_quantized.py in the future.

Reply:
I agree. Gonna create a low-pri task for myself